src/Microsoft.AspNetCore.Identity/SignInManager.cs

```diff
@@ -433,7 +433,19 @@ public virtual Task ForgetTwoFactorClientAsync()
         /// <param name="isPersistent">Flag indicating whether the sign-in cookie should persist after the browser is closed.</param>
         /// <returns>The task object representing the asynchronous operation containing the <see name="SignInResult"/>
         /// for the sign-in attempt.</returns>
-        public virtual async Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent)
+        public virtual Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent) 
```

Comment:
> This is a overload method for placing a default value for one parameter. Should we just have the one of the overload to be virtual? (I assume we can't change this in 1.1.0 since take away the virtual modifier will be a breaking change. But I'm just wondering.)
> 

I am not sure what the suggestion or ask is here. Can you clarify what you mean?